cmds/pg_unprivileged_replication/main.go

```diff
@@ -0,0 +1,126 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"flag"
+	"log"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/tomyl/pg-dump-upsert/pgdump"
+)
+
+func main() {
+	configFilePath := flag.String("config", "", "Path to configuraiton file (ex. config.json)")
+	startAt := flag.String("start-at", "",
+		"Manually force last-sync start time (useful when starting from a freshly-restored PGDUMP)")
+	verbose := flag.Bool("verbose", false, "Be verbose")
+	flag.Parse()
+
+	if configFilePath == nil || *configFilePath == "" {
+		log.Fatalln("The -config arguement is mandatory, see -help for details.")
+	}
+
+	var c config
+	if configFile, err := os.OpenFile(*configFilePath, os.O_RDONLY, 0); err != nil {
```

Comment:
don't forget to `defer configFile.Close()`

Reply:
Fair enough :p

Reply:
Closing it explicitly in line instead of with `defer` to avoid having the file stay open throughout the lifetime of the process ;)